File: 58-length-of-last-word/length-of-last-word.java
class Solution {
    public int lengthOfLastWord(String s) {
        String trimmedString = s.trim();
        int length = 0;
        for (int i = trimmedString.length() - 1; i >= 0; i--) {
            if (trimmedString.charAt(i) == ' ') {
                if (length > 0) {
                    break;
                }
            } else {
                length++;
            }
        }
        return length;
    }
}

File: 58-length-of-last-word/length-of-last-word.py
class Solution(object):
    def lengthOfLastWord(self, s):
        """
        :type s: str
        :rtype: int
        """
        ogString = s.strip()
        res = int(0)
        for i in ogString[::-1]:
            if i == ' ':
               break
            res += 1
        return res